static/app/types/project.tsx

```diff
@@ -67,6 +64,9 @@ export type Project = {
   verifySSL: boolean;
   builtinSymbolSources?: string[];
   defaultEnvironment?: string;
+  eventProcessing?: {
+    symbolicationDegraded?: boolean;
+  };
```

Comment:
it does look like it might always be there https://github.com/getsentry/sentry/blob/master/src/sentry/api/serializers/models/project.py#L1024-L1026

i wonder if the project was missing entirely from the store

Reply:
I wonder if it's related to this  - https://sentry.sentry.io/issues/5593998622/?project=11276. We might be loading something malformed into the projects store and causing these downstream errors

Reply:
the missing toSorted was an old browser thing

Reply:
welp nvm

Reply:
maybe when the project is created we use a different serializer?